### DISCUSSION TITLE
Kumquat vs Apple; a problematic answer on Unsupervised ML Quiz question.

### DISCUSSION THREAD

**Discussion by mynugraErlangga** [teamalgoritma/community#733] [Q&A] [answered]
Please take a look at the following cluster plot (using fviz):
![image](https://github.com/teamalgoritma/community/assets/125937571/98084b9c-2f7c-42d2-87b5-2acf2c5a6ab1)

On (red colored)cluster 1 notice the Apricot( mid-location), Kumquat (farthest-left), and Apple (far right).

Problem: The answer on the quiz Should be "Kumquat"(Which I selected) as a most-similar alternative to an Apricot given the options, instead of "Banana" or even "Apple.

Reasoning: Kumquat is closer in distance than Apple, relative to the Apricot.

Please consider, Team Algoritma. And fully upgrade my Quiz point

terima kasih

**Comment by divakartika** [comment, marked as answer]
Dear @mynugraErlangga 

Boleh dibantu untuk kirimkan file Rmd pengerjaan quiz nya ke Google Classroom, agar dapat kami cek proses pengerjaannya? Terima kasih.

Regards,
Team Algoritma

**Reply by kevbow** [reply]
Halo @mynugraErlangga , untuk menjaga agar forum diskusi tetap berjalan kondusif, kami izin melakukan lock/close diskusi ini. Semoga dapat dipahami.
